plugins/node/instrumentation-tedious/package.json

```diff
@@ -47,20 +47,23 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "6.21.0",
+    "@typescript-eslint/parser": "6.21.0",
+    "eslint": "8.9.0",
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/context-async-hooks": "^1.8.0",
     "@opentelemetry/contrib-test-utils": "^0.40.0",
     "@opentelemetry/sdk-trace-base": "^1.8.0",
     "@types/mocha": "7.0.2",
     "@types/node": "18.6.5",
-    "mocha": "7.2.0",
+    "mocha": "^10.4.0",
```

Comment:
is it needed for this PR to work?
we aim to align the entire repo to use the same common dependencies, so this should ideally be bumped as a dedicated PR that does that for all packages in the repo

Reply:
I understand. As far as I remember I had to update mocha because I had to replace mocha-ts with ts-node as described.

But I can double check if you want to 👌

Reply:
Thank you, @kirrg001.

I reviewed a few other instrumentations to see how they handle tests. Based on their dependencies, I believe our current setup should work without needing to bump the shared monorepo dependency versions unless absolutely necessary.

However, I do support the idea of opening a dedicated PR to update some of the dependencies across the repository. If you have the time and would like to contribute to this effort, it would be greatly appreciated.

Reply:
I will try to downgrade mocha to v7 again and let you know :)